source/AlarmWindow.cpp

```diff
@@ -283,13 +304,10 @@ bool AlarmWindow :: IsAfter(int min, int h, int d, int mon, int y) {
 	userTime = mktime(timeinfo);
 
 	// Compare user time and system time
-	if( difftime(userTime, time( &rawtime) ) > 0 ) {
-
+	if ( difftime(userTime, time( &rawtime) ) > 0 ) {
```

Comment:
No need for the curly brackets (plus the closing } farther down).

```suggestion
	if (difftime(userTime, time(&rawtime) ) > 0)
```